sandboxjs/fetch_sandbox.js

```diff
@@ -16,7 +16,6 @@
 
 const branchInfo = require('./bamboo_plan_info.json');
```

Comment:
is this still relevant?

Reply:
The version is still coming from there and our setup_gitpod.js uses it. Wanna confirm with Nick before deleting it.

Reply:
I'm pretty sure we don't use `setup_gitpod` anywhere but wouldn't hurt confirming.

Reply:
makes sense, thanks.